Sources/LiveViewNative/Coordinators/LiveSessionCoordinator.swift

```diff
@@ -326,6 +326,9 @@ public class LiveSessionCoordinator<R: RootRegistry>: ObservableObject {
         if domValues != nil {
             request.setValue(domValues.phxCSRFToken, forHTTPHeaderField: "x-csrf-token")
         }
+        for (header, value) in configuration.headers {
+            request.setValue(value, forHTTPHeaderField: header)
+        }
```

Comment:
Can we change the order these are applied?

```suggestion
        request.allHTTPHeaderFields = configuration.headers
        
        if domValues != nil {
            request.setValue(domValues.phxCSRFToken, forHTTPHeaderField: "x-csrf-token")
        }
```